events.ts

```diff
@@ -24,7 +24,7 @@ async function main() {
 
   if (contractId != null) {
     filters.push({
-      contractIds: [ new SorobanClient.Contract(contractId).contractId('hex') ]
+      contractIds: [ contractId ]
```

Comment:
we still want validation, right? I think we can/should pass `'strkey'` instead of `'hex'`

Reply:
Fixed.